build.sbt

```diff
@@ -149,7 +149,7 @@ lazy val httpCore = project("akka-http-core")
   .addAkkaModuleDependency("akka-stream", "provided")
   .addAkkaModuleDependency("akka-stream-testkit", "test")
   .addAkkaModuleDependency(
-    "akka-stream",
+    "akka-stream-testkit",
```

Comment:
ok, but should the above dependency to `akka-stream-testkit` still be there?

Reply:
Hmm good point, maybe it needs to be guarded with an opposite condition than the one below.

Reply:
Seems it is ok like that because the other line will be evicted once https://github.com/akka/akka-http/pull/2761 is merged.